src/main/java/org/openpnp/vision/pipeline/CvPipeline.java

```diff
@@ -51,16 +55,38 @@
     @ElementList
     private ArrayList<CvStage> stages = new ArrayList<>();
 
-    private Map<CvStage, Result> results = new HashMap<CvStage, Result>();
+    private Map<String, Result> results = new HashMap<String, Result>();
 
     private Mat workingImage;
 
     private Camera camera;
     private Nozzle nozzle;
     private Feeder feeder;
+    private Part	part;
     
     private long totalProcessingTimeNs;
-    
+
+	
+    /**************************************************************************
+     * Setter for internal flags
+     * Different Naming is used, because internal use only || checkstyle don't allow that
+     * Every flag have a prepending comment line that describes there use
+	 * Default value for flag must be false.
+     */
+	 
+	// indicate if camera is downlooking
+	private boolean downLooking;
+	// indicate if fiducial is searched
+	private boolean fiducial=true;
```

Comment:
Why? This is specific to one single algorithm and does not belong in CvPipeline.